.github/workflows/e2e.yml

```diff
@@ -0,0 +1,104 @@
+name: "Test: End to End"
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  end-to-end-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Workspace prefix
+        id: prefix
+        run: echo "::set-output name=name::action-e2e-test-$(date +%s)"
+      
+      - name: Workspace names
+        id: workspaces
+        env:
+          workspaces: '["alpha", "beta"]'
+        run: echo "::set-output name=names::${{ toJSON(env.workspaces) }}"
+
+      - name: Run action
+        uses: ./
+        with:
+          runner_terraform_version: '1.0.5'
+          terraform_organization: takescoop-oss
+          terraform_token: ${{ secrets.TF_TOKEN_OSS }}
+          name: ${{ steps.prefix.outputs.name }}
+          tags: |-
+            - terraform:true
+          workspace_tags: |-
+            alpha:
+              - workspace:alpha
+            beta:
+              - workspace:beta
+          apply: true
+          variables: |-
+            - key: foo
+              value: bar
+              category: terraform
+          workspace_variables: |-
+            alpha:
+              - key: workspace
+                value: alpha
+                category: terraform
+            beta:
+              - key: workspace
+                value: beta
+                category: terraform
+          workspaces: ${{ steps.workspaces.outputs.names }}
+          backend_config: |-
+            local:
+              path: /github/workspace/terraform.tfstate
```

Comment:
the issue here is that the working directory is different within the docker env from running bash commands in subsequent steps

`github.workspace` resolves to `/home/runner/work/<repo>/<repo>` which is mapped to `/github/workspace`, so we can't template that in at the action level.

Reply:
Ok, mention that in a comment